core/src/main/java/org/polypheny/db/sql/ddl/SqlCreateView.java

```diff
@@ -154,9 +153,10 @@ public void execute( Context context, Statement statement ) {
                     relCollation,
                     replace,
                     statement,
-                    store,
                     placementType,
-                    columns );
+                    columns,
+                    query.toString().replaceAll( "`", "\"" ),
```

Comment:
This looks strange

Reply:
Query needs to be with quotation marks, now using method to get correct query.